src/popup/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -372,102 +366,61 @@ export default {
     }
   }
 
-  .token-header {
-    background-color: variables.$color-black;
+  ::v-deep .tokens {
```

Comment:
Yes, I added `::v-deep` here. This is temporary because this whole screen will be rewritten. So I just needed to fix the page after removing the `<Plate>` component.